### DISCUSSION TITLE
Instalation on Unraid

### DISCUSSION THREAD

**Discussion by zapa11** [Security-Onion-Solutions/securityonion#11640] [2.4]
### Version

2.4.0

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

8

### Storage for /

100

### Storage for /nsm

200

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Hello,

I am trying to install SecurityOnion on Unraid . After i boot the VM, i end up in Grub.
Any tips how to configure please?

Thanks,

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
Did you verify the ISO image as shown at https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/main/DOWNLOAD_AND_VERIFY_ISO.md?

Also, please note that 8GB RAM is sufficient for selecting the IMPORT installation but does not meet the minimum requirements for the STANDALONE option:
https://docs.securityonion.net/en/2.4/hardware.html#standalone-deployments
